back/spec/acceptance/projects_allowed_input_topics_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+
+resource "ProjectsAllowedInputTopics" do
+
+  explanation "E.g. mobility, health, culture..."
+
+  before do
+    header "Content-Type", "application/json"
+    @projects_allowed_input_topics = create_list(:projects_allowed_input_topic, 2)
+  end
+
+  # get "web_api/v1/projects_allowed_input_topics" do
+  #   with_options scope: :page do
+  #     parameter :number, "Page number"
+  #     parameter :size, "Number of topics per page"
+  #   end
+    
+  #   example_request "List all projects_allowed_input_topics" do
+  #     expect(status).to eq(200)
+  #     json_response = json_parse(response_body)
+  #     expect(json_response[:data].size).to eq 2
+  #   end
+  # end
```

Comment:
@sebastienhoorens I am thinking of deleting this spec, since I can't think when it would make sense to list all `allowed_input_topics` for all projects.

I believe this PR also changes the index route to avoid this possibility in any case.